exporters/elasticsearch-exporter/src/test/java/io/zeebe/exporter/util/ElasticsearchContainer.java

```diff
@@ -152,27 +148,22 @@ protected void configure() {
   }
 
   private void setupUser() {
-    final User user = new User(username, Collections.singleton("zeebe-exporter"));
+    final var request = new Request("POST", "/_xpack/security/user/" + username);
+    final var body = Map.of("roles", Collections.singleton("zeebe-exporter"), "password", password);
```

Comment:
Even though this is just a test class, we should set a good example and use `password.toCharArray()` here instead of password (as a String), to set a good example of how to deal with passwords in Java code. The array should also be cleared from memory after using.

Reply:
The Elasticsearch API expects a string there, and if I use `toCharArray`, wouldn't the JSON mapper serialize it as an actual array?

Reply:
Yeah, I'm curious whether it works. If not, I think it's totally fine to just use a String here.

Reply:
It does work ;)